src/ert/config/queue_config.py

```diff
@@ -20,11 +19,53 @@
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
 VALID_QUEUE_OPTIONS: Dict[Any, List[str]] = {
-    QueueSystem.TORQUE: _clib.torque_driver.TORQUE_DRIVER_OPTIONS
-    + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LOCAL: [] + GENERIC_QUEUE_OPTIONS,  # No specific options in driver
-    QueueSystem.SLURM: _clib.slurm_driver.SLURM_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LSF: _clib.lsf_driver.LSF_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
+    QueueSystem.TORQUE: GENERIC_QUEUE_OPTIONS
+    + [
+        "CLUSTER_LABEL",
+        "DEBUG_OUTPUT",
+        "JOB_PREFIX_KEY",
```

Comment:
My list has `JOB_PREFIX`

Reply:
thanks! (there is yet another set of names in the hpp file which are the names of the constants..)

Reply:
I find those files v confusing!